app/controllers/application_controller.rb

```diff
@@ -284,18 +284,12 @@ def unsafe_redirect_error(_exception)
   end
 
   def user_fully_authenticated?
-    !reauthn? &&
-      user_signed_in? &&
+    user_signed_in? &&
       session['warden.user.user.session'] &&
       !session['warden.user.user.session'][TwoFactorAuthenticatable::NEED_AUTHENTICATION] &&
       two_factor_enabled?
   end
 
-  def reauthn?
-    reauthn = reauthn_param
```

Comment:
I think `reauthn_param` is unused now, so we could remove the method.

It's also overridden in `TwoFactorAuthenticationController`. I assume the specific override logic is accounted for in the revisions there?

https://github.com/18F/identity-idp/blob/ba0bdd0d82501b184e467f3edfc4a19c1d252c51/app/controllers/users/two_factor_authentication_controller.rb#L173-L176

Reply:
ahh, yep, will remove